spec/actions.array.lsl

```diff
@@ -0,0 +1,64 @@
+//#! pragma: non-synthesizable
+libsl "1.1.0";
+
+library any
+    version "*"
+    language "any"
+    url "-";
+
+
+// === LIFE-TIME management ===
+
+
+define action ARRAY_NEW(
+        typeName: string, // literal!
+        size: int32
+    ): array<any>;
+
+// unused
+define action ARRAY_FREE(
+        arr: array<any>
+    ): void;
+
+
+// === READ operations ===
```

Comment:
We can use `LOOP_*` actions as there are currently no optimized (intrinsic) versions of the same *action*: loop through an array to find a matching item while returning the index of one or `-1`.
If there will be a necessity in making this its own action, then, yes, it would help (less copy-paste).